packages/manager/modules/pci/src/components/project/storages/databases/pool.class.js

```diff
@@ -0,0 +1,33 @@
+export default class Pool {
+  constructor({
+    databaseId,
+    databaseName,
+    id,
+    mode,
+    name,
+    size,
+    uri,
+    userId,
+    userName,
+  }) {
+    Object.assign(this, {
+      databaseId,
+      databaseName,
+      id,
+      mode,
+      name,
+      size,
+      uri,
+      userId,
+      userName,
+    });
+  }
+
+  setUserName(user) {
```

Comment:
Nitpick: you can use ES6 setters and getters like set userName() and set databaseName(). These methods will be called when userName or databaseName fields are referenced.

Reply:
Hi Ravi, i tried to do it but we need to give different names to use `setter` as it calls setter again. This definitely looks cleaner than using setters. For `get` it would be good though, not for `set`